internal/cloudproviders/aws/aws_client.go

```diff
@@ -40,7 +61,7 @@ const (
 	initialNicDeletionWaiterTime   = time.Second * 30
 	instanceInitialTerminationTime = time.Second * 200
 	managedClusterActiveWaiter     = time.Minute * 10
-	managedNodeGroupActiveWaiter   = time.Minute * 10
+	managedNodeGroupActiveWaiter   = time.Minute * 4
```

Comment:
This is interesting!
is it a linear backoff or a exponential backoff's inital time?